docs/ops/arithmetic/Add_1.md

```diff
@@ -6,46 +6,48 @@
 
 **Short description**: *Add* performs element-wise addition operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Add* performs addition operation for the input tensors *a* and *b* using formula below:
+
+\f[
+o_{i} = a_{i} + b_{i}
+\f]
+
 **Attributes**:
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes must match
+    * *numpy* - numpy broadcasting rules, description is available in <a href="https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Broadcast_1.md">Broadcast op</a>.
```

Comment:
Refer to other operation like this   `[Broadcast_1](../movement/Broadcast_1.md)`

Reply:
Corrected.